UITests/UITests.Tests.Shared/UITestBase.cs

```diff
@@ -69,156 +66,81 @@ public static TestApplicationInfo WinUICsUWPSampleApp
             }
         }
 
-        public static TestSetupHelper.TestSetupHelperOptions TestSetupHelperOptions
-        {
-            get
+        private static TestSetupHelper.TestSetupHelperOptions TestSetupHelperOptions
+            => new()
             {
-                return new TestSetupHelper.TestSetupHelperOptions
-                {
-                    AutomationIdOfSafeItemToClick = string.Empty
-                };
-            }
-        }
+                AutomationIdOfSafeItemToClick = string.Empty
+            };
 
         public TestContext TestContext { get; set; }
 
-        private AppServiceConnection CommunicationService { get; set; }
-
         [TestInitialize]
-        public async Task TestInitialize()
+        public void TestInitialize()
         {
-            PreTestSetup();
+            // This will reset the test for each run (as from original WinUI https://github.com/microsoft/microsoft-ui-xaml/blob/master/test/testinfra/MUXTestInfra/Infra/TestHelpers.cs)
+            // We construct it so it doesn't try to run any tests since we use the AppService Bridge to complete
+            // our loading.
+            helper = new TestSetupHelper(new string[] { }, TestSetupHelperOptions);
 
-#if USING_TAEF
-            var fullTestName = TestContext.TestName;
-            var lastDotIndex = fullTestName.LastIndexOf('.');
-            var testName = fullTestName.Substring(lastDotIndex + 1);
-            var theClassName = fullTestName.Substring(0, lastDotIndex);
-#else
-            var testName = TestContext.TestName;
-            var theClassName = TestContext.FullyQualifiedTestClassName;
-#endif
-            var currentlyRunningClassType = AppDomain.CurrentDomain.GetAssemblies().SelectMany(a => a.GetTypes()).FirstOrDefault(f => f.FullName == theClassName);
-            if (!(Type.GetType(theClassName) is Type type))
-            {
-                Verify.Fail("Type is null. TestClassName : " + theClassName);
-                return;
-            }
+            var pageName = GetPageForTest(TestContext);
 
-            if (!(type.GetMethod(testName) is MethodInfo method))
-            {
-                Verify.Fail("Mothod is null. TestClassName : " + theClassName + " Testname: " + testName);
-                return;
-            }
+            var rez = TestAssembly.OpenPage(pageName);
+            rez.Wait();
```

Comment:
Why not leave this TestInitialize method as async and do the await here?

Reply:
I think I changed the method because there were issues having an `async` `TestInitalze` method. I made a lot of changes so I can try as an `async` method again.

Reply:
@RosarioPulella did this have to do with how initialize clean-up was being performed when async?

Reply:
It seems to work the same, I made the change.